test/Database/LMDB/Simple/DBRefSpec.hs

```diff
@@ -8,24 +9,25 @@ import Harness
 import Test.Hspec
 
 spec :: Spec
-spec = beforeAll (setup >>= \(env, db) -> newDBRef env db 0) $ do
+spec = beforeAll (setup "DBRefSpec" >>= \(env, db, cleanup) -> (env,,cleanup) <$> newDBRef env db 0) $
```

Comment:
`(env,,cleanup)`
what is this kind of magic 😲

Reply:
`{-# LANGUAGE TupleSections #-}`!